packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -194,6 +202,15 @@ function Table<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<HTMLD
           return <TableSelectAllCell column={item} />;
         }
 
+        if (item.props.noHeader) {
+          return (
+            <TooltipTrigger delay={0} placement="top">
```

Comment:
from the pictures, it looks pretty far, maybe we should reduce the offset from the tooltip? though then it may not default align with tooltips on the columns with headers...
is there a design for this? just curious

Reply:
![image](https://user-images.githubusercontent.com/30304082/93828599-90665500-fc20-11ea-8cf8-2b86b3d16cca.png)
This what I got, let me update my screenshot with focused header

Reply:
well, there is still a case where there is no focus ring right? it appears on hover too?

Reply:
tooltips are only on columns without headers. Do you think it would make sense to move the tooltip a bit closer and serve both the focus and hover state OR should we set different offsets based on whether the headerless column header is being focused or hovered?

Reply:
@snowystinger Added offset as suggested.